yt/data_objects/construction_data_containers.py

```diff
@@ -741,12 +741,15 @@ def deposit(self, positions, fields = None, method = None,
         cls = getattr(particle_deposit, "deposit_%s" % method, None)
         if cls is None:
             raise YTParticleDepositionNotImplemented(method)
-        # We allocate number of zones, not number of octs
-        op = cls(self.ActiveDimensions, kernel_name)
+        # We allocate number of zones, not number of octs. Everything inside
+        # this is fortran ordered because of the ordering in the octree deposit
+        # routines, so we reverse it here to match the convention there
+        op = cls(tuple(self.ActiveDimensions)[::-1], kernel_name)
```

Comment:
So if I understand correctly, this is actually the issue, right?  If we don't do this, everything else is wrong, even if the arrays are ordered correctly?  (I'm just trying to dissect the `[::-1]` bit.)